src/components/Range/index.tsx

```diff
@@ -291,15 +328,20 @@ const Range = ({
     [ceil, onChange, setInternValues, getNextValues],
   )
 
-  const onMouseDown = useCallback(index => {
+  const onMouseDown = useCallback((index: number) => {
     grabCursor(index)
   }, [])
 
   const onMouseMove = useCallback(
-    ev => {
-      if (ev.target.tagName === 'INPUT') return
-      const cursor = cursorsRef[grabbedCursor].current
-      if (cursor) {
+    (ev: MouseEvent) => {
+      if ((ev.target as HTMLElement).tagName === 'INPUT') return
+
+      let cursor
```

Comment:
```suggestion
      const cursor = cursorsRef?.[grabbedCursor]?.current 
```

Would be the same ?

Reply:
Nope: Type 'undefined' cannot be used as an index type.